Makefile

```diff
@@ -125,10 +119,36 @@ dv_review_aks: ## make dv_review_aks deploy APP_NAME=2222 CLUSTER=cluster1
 	$(eval export TF_VAR_app_name=$(APP_NAME))
 	$(eval export TF_VARS=-var config_short=${CONFIG_SHORT} -var service_short=${SERVICE_SHORT} -var service_name=${SERVICE_NAME} -var azure_resource_prefix=${RESOURCE_NAME_PREFIX})
 	echo https://register-$(APP_NAME).$(CLUSTER).development.teacherservices.cloud will be created in aks
-#	$(eval APP_NAME_SUFFIX=dv-review-$(APP_NAME))
-#	$(eval backend_key=-backend-config=key=pr-$(APP_NAME).tfstate)
-#	$(eval export TF_VAR_app_name_suffix=pr-$(APP_NAME))
 
+qa_aks:
+	$(eval include global_config/qa_aks.sh)
+	$(eval DEPLOY_ENV=qa_aks)
+	$(eval DTTP_HOSTNAME=traineeteacherportal-dv)
+	$(eval paas_env=qa)
```

Comment:
Should this be here? Should there be one in staging as well if so?

Reply:
removed as not required for aks env